### PR TITLE
net-analyzer/hexinject: Bump to 1.6 and package occupation.

### PR DESCRIPTION
**Pull request by holgersson32644**
Package-Manager: Portage-2.3.11, Repoman-2.3.3

### CODE REVIEW

File: net-analyzer/hexinject/metadata.xml [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+	<maintainer type="person">
+		<email>holgersson@posteo.de</email>
+		<name>Nils Freydank</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
```
Comment (jonasstein) [inline net-analyzer/hexinject/metadata.xml]: indent

File: net-analyzer/hexinject/hexinject-1.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Network packet sniffer and injector"
+HOMEPAGE="http://hexinject.sourceforge.net/"
+SRC_URI="http://downloads.sourceforge.net/project/${PN}/${P}/${P}.tar.gz"
```
Comment (Whissi) [inline net-analyzer/hexinject/hexinject-1.6.ebuild]: SourceForge has a special mirror handling support in Gentoo. Please use *mirror://sourceforge* notation.

**Comment by Whissi** [comment]
Partially merged (metadata.xml only). Package has multiple QA issues which we need to address:

* Ebuild is calling *gcc* directly (https://bugs.gentoo.org/243502):
```
>>> Compiling source in /var/tmp/portage/net-analyzer/hexinject-1.6/work/hexinject ...
make --jobs 6
gcc -Wall -lpcap -o hexinject hexinject.c
```

* Ebuild doesn't respect flags
```
>>> Compiling source in /var/tmp/portage/net-analyzer/hexinject-1.6/work/hexinject ...
make --jobs 6
gcc -Wall -lpcap -o hexinject hexinject.c
gcc -Wall -lpcap -o prettypacket prettypacket.c
gcc -Wall -lpcap -o hex2raw hex2raw.c
>>> Source compiled.
>>> Test phase: net-analyzer/hexinject-1.6
>>> Completed testing net-analyzer/hexinject-1.6

>>> Install hexinject-1.6 into /var/tmp/portage/net-analyzer/hexinject-1.6/image/ category net-analyzer
>>> Completed installing hexinject-1.6 into /var/tmp/portage/net-analyzer/hexinject-1.6/image/

 * Final size of build directory: 204 KiB
 * Final size of installed tree:  116 KiB


 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/hex2raw
 * /usr/bin/hexinject
 * /usr/bin/prettypacket
````

**Comment by Whissi** [comment]
Collided with https://github.com/gentoo/gentoo/pull/5931.

I'll rev bump with the QA fixes we worked on.

**Pull request closed without merging**